README.md

```diff
@@ -34,8 +34,6 @@ All essential commands are available at the root via `npm run <script name>` - e
 <summary><b>How to run your tests</b></summary>
 <p>
 
-The `Agent` integration tests ([source](./test/Agent.integration.js)) depend on a real HTTP api backend and a valid Particle access token. Be sure to set relevant environment variables to avoid test failures. You can prefix commands test commands like this `PARTICLE_API_BASE_URL=<url> PARTICLE_API_TOKEN=<token> npm test`
-
```

Comment:
@joegoggins 

>Also, eliminates problem where PARTICLE_API_TOKEN had to be specified for many npm run commands by moving *.js files that require these to an e2e-tests that does NOT run in CI. This is not a step down in terms of quality; as these tests were not designed to be run in this CI context, but rather manually by developers working on the event stream.

fwiw, @deleonn updated CI over in https://github.com/particle-iot/particle-api-js/pull/142 to ensure at least the node-side event e2e tests ran with their required perquisites - here's an example of them running and "passing":

https://app.circleci.com/pipelines/github/particle-iot/particle-api-js/53/workflows/cd0aa933-14fd-4836-a96b-024f13e31457/jobs/156?invite=true#step-103-42

further complicating things, the reference to "`Agent` integration tests" in these docs is incorrect - it's actually the `test/EventStream-e2e-node.js` file which is running in CI.

all that said, at this point i'd just delete all of those (`e2e-tests` directory and related) as they're likely to cause more confusion than help.